library/globals.inc.php

```diff
@@ -175,9 +175,9 @@ function gblTimeZones()
             xl('Pick a general theme (need to logout/login after change this setting).')
         ),
 
-        'font-family' => [
```

Comment:
I Looked into this and this is supported in PHP 5.4 and above(note 5.3 php version is very old and has not been supported for some time and rec avoiding using it). OpenEMR 5.0.1 will require 5.6+, so this should not be an issue.